### DISCUSSION TITLE
Git: remote: Repository not found.

### DISCUSSION THREAD

**Discussion by ft-Basil** [community/community#45993] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi,
I was trying to work with git from my vscode. Tried making a repo on github and tried to connect it through vscode, it throws error repo not found. I tried vice versa creating from vs code. repo was created but when i try to push it throws repo not found. I tried changed remote, renaming it, creating anther branch, tried making another repo, but the result is same!

<img width="278" alt="Screenshot 2023-02-01 at 2 17 17 AM" src="https://user-images.githubusercontent.com/122949036/215976675-28e6b9a9-5448-4c60-9f46-daff4c530c29.png">

when I tried on clicking the link on terminal, it directs me to correct repo!!

<img width="701" alt="Screenshot 2023-02-01 at 2 20 52 AM" src="https://user-images.githubusercontent.com/122949036/215977289-d1cf5f07-37b1-48ef-9140-8edd94275f8e.png">

**Comment by jge162** [comment, marked as answer]
Hello and good day @ft-Basil issue could be due to the incorrect remote repository URL being used in Visual Studio Code. Run->`Git: Remotes`

If the remote repository URL is incorrect, you can update it by opening the Command Palette and running the command `Git: Add Remote.` Enter a name for the remote and the correct repository URL, then run the command `Git: Push` to push the changes to the correct remote repository.

also check you credentials -> run `Git: Credential Manager` do they look correct?

**Reply by ft-Basil** [reply]
Thanks @jge162 Did `Git: Credential Manager` it worked <3

**Reply by jge162** [reply]
Perfect. If you’re happy with answer can you update posts please. It will help someone else if they search this issue in the future. Thanks again.

**Reply by gregg-cbs** [reply]
In the terminal i had to type `gh auth login`, went through the prompts and pasted my personal access token and the resolved my issue. I was logged in yesterday, my token hasnt expired but for some reason it stopped working.
